warden/jwt.go

```diff
@@ -0,0 +1,63 @@
+package warden
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	auth0 "github.com/auth0-community/go-auth0"
+	"github.com/gin-gonic/gin"
+	jose "gopkg.in/square/go-jose.v2"
+	jwt "gopkg.in/square/go-jose.v2/jwt"
+)
+
+func verifyJWT(request *http.Request) (*jwt.Claims, error) {
+	domain := request.Header.Get("Auth0-Domain")
```

Comment:
Should this be part of the config of the instance? Probably read from the ENV variables?

Reply:
Maybe for this one indeed, but not for the audience right? (each Relying Party like Balrog have their own audience, unless I'm mistaken...)

Reply:
Yes then would we be able to grab it from the referer?

Reply:
The caller (Relying Party, like balrog) is not a browser, so it has to send it explicitly